core/libraries/form_sections/base/EE_Form_Section_Proper.form.php

```diff
@@ -378,7 +378,7 @@ protected function get_submitted_form_data_from_session()
      * @throws InvalidInterfaceException
      * @throws InvalidDataTypeException
      */
-    protected function flush_submitted_form_data_from_session()
+    public static function flush_submitted_form_data_from_session()
```

Comment:
why static?

does the action below not work correctly if it were:

```php
add_action(
    'shutdown', 
    array($this, 'flush_submitted_form_data_from_session')
);
```

Reply:
good question. Because if there's a form with sub-forms, each one will set this as a callback from `EE_Form_Section_Proper::populate_from_session()` as that gets called on each sub-section. If this weren't static, then we'd call this method again for each sub-form which seems unnecessary. 
Granted, we could also achieve this by leaving it non-static, and making sure we only call it for formsections that have no parents. Would you prefer that?

Reply:
no that's a valid use of static imho. good call.